Web/starter_project_vanguard/components/profile/PersonalInfo.tsx

```diff
@@ -0,0 +1,120 @@
+import Image  from 'next/image'
+import React from 'react'
+
+import {User} from '@/types/user-profile'
+import profile from 'public/images/profile-photo.png'
+
+const user: User = {
+  firstName: 'Segni',
+  lastName: 'Desta',
+  email: 'segni@gmail.com',
+  img: profile,
+}
```

Comment:
Even if it's small, I would rather not have mock data here. It would be better to separate it.

Reply:
In which folder do you think I should put it? (store is for redux, i guess)